test/demo/createCenter.inttest.ts

```diff
@@ -1,4 +1,5 @@
 import 'jest';
+import 'reflect-metadata';
```

Comment:
Can you explain this?

Reply:
This looks to be left over from either testing.  At the root its use is for injection stuff.  I'll remove it

Reply:
Removed